index.html

```diff
@@ -3,8 +3,9 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" href="path/to/output.css" />
```

Comment:
This was supposed to link to a separate style sheet created called output.css It's no longer needed since I did not update yet. I can remove this line of code as well